spec/models/service_plan_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe ServicePlan do
+  describe "#order" do
+    let(:service_plan) do
+      described_class.new(:id               => 456,
```

Comment:
Shouldn't need to specify an ID here, it'll be set automatically

Reply:
Oh yeah, it'll be set automatically when you call `.create!` :) prefer that over `.new`